hedera-node/hedera-evm/src/test/java/com/hedera/node/app/service/evm/contracts/operations/HederaEvmCreateOperationTest.java

```diff
@@ -71,10 +75,29 @@ void computesExpectedCost() {
 
     @Test
     void computesExpectedTargetAddress() {
-        given(frame.getWorldUpdater()).willReturn(hederaWorldUpdater);
+        given(frame.getWorldUpdater()).willReturn(stackedUpdater);
         given(frame.getRecipientAddress()).willReturn(recipientAddr);
-        given(hederaWorldUpdater.newContractAddress(recipientAddr)).willReturn(Address.ZERO);
+        given(stackedUpdater.get(recipientAddr)).willReturn(sender);
+        given(sender.getNonce()).willReturn(1L);
```

Comment:
(BTW, this is why I really dislike mocking.  Here we're mocking down into the deep data structure of `frame.getWorldUpdater().get(recipientAddr).getNonce()` where `recipientAddr` is _also_ mocked.  Just seems fragile - working with internal details you shouldn't have to in order to do a test.  Not sure in this codebase how it should have been done differently so this kind of mocking in unit tests wouldn't be necessary - but _something_ is a bit off ....)